app/components/modal/publicCloudProductDelete.tsx

```diff
@@ -122,7 +129,11 @@ export const openPublicCloudProductDeleteModal = createModal<ModalProps, ModalSt
                   classNames={{ wrapper: 'mt-1' }}
                 />
                 <HookFormTextInput name="email" placeholder="Product owner email" classNames={{ wrapper: 'mt-1' }} />
-
+                <HookFormTextarea
```

Comment:
let's add a label to display the description on top of the textarea.
`Please specify the reason for deleting this product.`